### PR TITLE
Qualification form

### LINKED ISSUE #76: Integrate Qualification "Details" form (1/5)
required by #75

### LINKED ISSUE #77: Integrate Qualification "Duration and fees" form (2/5)
required by #75

### PR DESCRIPTION
**Pull request by dimasciput**
Close https://github.com/kartoza/ford3/issues/76, close https://github.com/kartoza/ford3/issues/77, partially integrate https://github.com/kartoza/ford3/issues/78

### CODE REVIEW

**Comment by dimasciput** [comment]
ford

File: django_project/core/urls.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def handler500(request, template_name='500.html', *args, **kwargs):
     url(r'^site-admin/', admin.site.urls),
     url(r'^ford3/', include('ford3.urls')),
     url(r'^', include('base.urls')),
-    # url(r'^grappelli/', include('grappelli.urls')),
+    url(r'^grappelli/', include('grappelli.urls')),
```
Comment (lucernae) [inline django_project/core/urls.py]: Maybe we should move this up so base urls are resolved last.

File: django_project/ford3/models/requirement.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+from ford3.enums.saqa_qualification_level import SaqaQualificationLevel
 
 
 class Requirement(models.Model):
     qualification_id = models.ForeignKey(
         Qualification,
         on_delete=models.CASCADE)
 
-    id = models.IntegerField(
+    id = models.AutoField(
```
Comment (lucernae) [inline django_project/ford3/models/requirement.py:11]: I just realized we have an id field.
Why we need id field explicitly?
Wasn't this handled in the background by Django?

**Review by lucernae — APPROVED** [review]
LGTM.
I think we can ignore the coverage in this PR.
To increase coverage, I think it is better to do that in a separate PR to include unittests.

**Pull request merged**